apps/inbound-mail/src/.env.test

```diff
@@ -1,11 +1,23 @@
 NODE_ENV=test
 
-POST="25"
+PORT="2500"
```

Comment:
very strange that it fails on ports under 1024

Reply:
No, it is not strange. It is a Unix security feature for non-root processes.